docparse/jsonschema.go

```diff
@@ -573,3 +557,24 @@ func canonicalType(currentFile, pkgPath string, typ *ast.Ident) (ast.Expr, error
 
 	return ts.Type, nil
 }
+
+func readAndUnmarshalSchemaFile(path string, target interface{}) error {
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		return fmt.Errorf("could not read file %q: %v", path, err)
+	}
+
+	var f func([]byte, interface{}) error
+	switch strings.ToLower(filepath.Ext(path)) {
+	default:
+		return fmt.Errorf("unknown file type: %q", path)
+	case ".json":
+		f = json.Unmarshal
+	case ".yaml":
```

Comment:
maybe add `.yml` as well here? 🤔

Reply:
This project only accepts the one true file extension, I think it isn't supported anywhere I'll have a look

Reply:
Fair enough, `.yaml` is the official one alright 😊

Reply:
Includ~1 support for long deprec~1 operat~1 systems is not a priority?